src/components/Homepage/home.js

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchCases } from '../../redux/covidCases/cases';
+import Filter from './Homeitems/Filter';
+import Item from './Homeitems/Item';
+import Nav from '../Nav';
+import covid from './covid.svg';
+
+const Home = () => {
+  const covidData = useSelector((state) => state.covidReducer);
+  const dispatch = useDispatch();
+
+  let countDesign = 0;
+  let switcher = 0;
+  let color1;
+
+  const [worldData, setWorldData] = useState(0);
+
+  const today = new Date().toISOString().slice(0, 10);
+
+  const URL = 'https://api.covid19tracking.narrativa.com/api';
+
+  const fetchData = async () => {
+    const response = await fetch(`${URL}/${today}`, {
+      method: 'GET',
+    });
+    const jsonRespnse = await response.json();
+    const neededData = jsonRespnse.dates[`${today}`].countries;
+    setWorldData(jsonRespnse.total);
+    dispatch(fetchCases(Object.values(neededData)));
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <div className="container-fluid " style={{ backgroundColor: '#FC5193' }}>
+      <div className="row">
+        <Nav year="2022" title="Whole World Data" />
+        <Filter data={worldData} />
+        <p className="text-white m-0 p-1" style={{ backgroundColor: '#DD4883' }}><b>STATS BY COUNTRY</b></p>
+        {
+          /* eslint-disable array-callback-return */
+          /* eslint-disable consistent-return */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.